be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -235,7 +235,8 @@ class LakeTabletsChannel : public TabletsChannel {
     mutable bthreads::BThreadSharedMutex _rw_mtx;
     std::unordered_map<int64_t, uint32_t> _tablet_id_to_sorted_indexes;
     std::unordered_map<int64_t, std::unique_ptr<AsyncDeltaWriter>> _delta_writers;
-    std::unique_ptr<BundleWritableFileContext> _bundle_writable_file_context;
+    // Partition id -> BundleWritableFileContext
+    std::unordered_map<int64_t, std::unique_ptr<BundleWritableFileContext>> _bundle_writable_file_contexts;
```

Comment:
```suggestion
    std::unordered_map<int64_t, std::unique_ptr<BundleWritableFileContext>> _bwfile_ctx_by-part;
```
Just a suggestion. prefer naming a map with a suffix `_by_xxx`.